website/source/docs/commands/remote-config.html.markdown

```diff
@@ -50,6 +50,11 @@ The following backends are supported:
   variables can optionally be provided. Address is assumed to be the
   local agent if not provided.
 
+* Etcd - Stores the state in etcd at a given path.
+  Requires the `path` and `endpoints` variables. The `username` and `password`
+  variables can optionally be provided. `endpoints` is assumed to be a
+  comma-separated list of etcd endpoints.
```

Comment:
I considered spaces, I went for commas mainly because handling arguments with spaces in them in shell scripts is something people tend to mess up. Commas seemed rare enough in URLs for this to be worth it. I'm going to change this to spaces, though — two votes for spaces, and only one for commas.